lib/packwerk/package_set.rb

```diff
@@ -31,17 +31,26 @@ def load_all_from(root_path, package_pathspec: nil)
         new(packages)
       end
 
-      sig { params(root_path: String, package_pathspec: T.any(String, T::Array[String])).returns(T::Array[Pathname]) }
-      def package_paths(root_path, package_pathspec)
-        bundle_path_match = Bundler.bundle_path.join("**").to_s
+      sig do
+        params(
+          root_path: String,
+          package_pathspec: T.any(String, T::Array[String]),
+          exclude_pathspec: T.any(String, T::Array[String])
```

Comment:
I'm wondering if we should introduce a type alias (or even a type) for path specs at this point